srcpkgs/juliaup/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'juliaup'
+pkgname=juliaup
+version=1.8.16
+revision=1
+archs="x86_64* i686* aarch64*"
```

Comment:
I only see musl downloads available for x86_64: https://julialang.org/downloads/

Reply:
Thanks for pointing that out. Fixed.